pakyow-core/lib/core/middleware/req_path_normalizer.rb

```diff
@@ -0,0 +1,27 @@
+require 'core/call_context'
+
+module Pakyow
```

Comment:
@bryanp I added the template for it in `lib/generators/pakyow/app/templates/middleware/req_path_normalizer.rb`, do I need to add something to the `app_generator.rb` to make sure it gets added to `setup.rb` when the app gets generated?